tests/test_utils.py

```diff
@@ -196,7 +196,8 @@ def reset(self, **kwargs):
 def test_evaluate_policy_monitors(vec_env_class):
     # Test that results are correct with monitor environments.
     # Also test VecEnvs
-    n_eval_episodes = 2
+    n_eval_episodes = 3
```

Comment:
Please also add a test that check that the number of episodes is the one expected and make `n_envs` a parameter of the test function.
You should also add a test for the evaluation callback with multiple envs.

Reply:
I added two tests for these cases.